docs/src/user-docs/guides-using-presto.md

```diff
@@ -33,27 +33,55 @@ Using Presto with CLP requires:
 
 1. Follow the [quick-start](quick-start/index.md) guide to download and extract the CLP package,
    but don't start the package just yet.
-2. Before starting the package, update the package's config as follows:
+2. Before starting the package, update the package's config file (`etc/clp-config.yml`) as follows:
 
-    * Open `etc/clp-config.yml` located within the package.
-    * Uncomment the `database` section.
-    * Change `database.host` value to a non-localhost hostname/IP.
-    * After the change, the `database` section should look something like this:
+    * Set the `package.query_engine` key to `"presto"`.
 
       ```yaml
-      database:
-        type: "mariadb"  # "mariadb" or "mysql"
-        host: "<new-IP-address>"
-        port: 3306
-        name: "clp-db"
+      package:
+        storage_engine: "clp-s"
+        query_engine: "presto"
       ```
 
-    :::{note}
-    This change is necessary since the Presto containers run on a Docker network, whereas CLP's
-    database runs on the host network. So `localhost` refers to two different entities in those
-    networks. This limitation will be addressed in the future when we unify Presto and CLP's
-    deployment infrastructure.
-    :::
+    * Set the `database.host` key to a non-localhost hostname/IP.
+
+      ```yaml
+        database:
+        #  type: "mariadb"
+          host: <IP-address>
+        #  port: 3306
+        #  name: "clp-db"
+      ```
+
+      :::{note}
+      This change is necessary because the Presto containers run on a Docker network, and CLP's
+      database runs on the host network. `localhost` will refer to a different entity in each of
+      those contexts.
+      :::
+
+    * Set the `results_cache.retention_period` key to `null`. The CLP + Presto integration does not
+      yet provide support for garbage collection.
+
+      ```yaml
+      results_cache:
+        #  host: "localhost"
+        #  port: 27017
+        #  db_name: "clp-query-results"
+        #  stream_collection_name: "stream-files"
+        #
+        #  # Retention for search results, in minutes. Set to null to disable automatic deletion.
+          retention_period: null
+      ```
+
+    * Update the `presto` key with the host and port of the Presto cluster. If you follow the
+      [Setting up Presto](#setting-up-presto) guide, the host is `localhost` and the port is `8889`.
+      The Presto cluster does not need to be running to start the package.
+
+      ```yaml
+      presto:
+        host: <IP-address>
+        port: <port>
+      ```
```

Comment:
```suggestion
      ```

      :::{note}
      Presto doesn't need to be running before you start CLP.
      :::
```